packages/mjml-image/src/index.js

```diff
@@ -34,6 +34,8 @@ export default class MjImage extends BodyComponent {
     height: 'unit(px,auto)',
     'max-height': 'unit(px,%)',
     'font-size': 'unit(px)',
+    'custom-attr-a': 'string',
```

Comment:
And then we add `custom-attr-tr`, `custom-attr-td`, custom-attr-XX, this won't really scale really much :/

Reply:
custom-attr-[tag] will allow user to specify which tag to attach the attr to, which gives the user some control over it. Adding attr to every tag is probably not necessary.

Reply:
There's a lot of HTML produced by a single tag an the current solution would require to do for every single html element output by mjml, even (and for like... section it would be hard to do so because you have multiple table/tr/td/....).

Reply:
The point is, in most cases, developer would add custom attrs to a/img/p tags, table/tr/td are for getting the view display right for different mail clients, so I think custom attr can be supported for a selected list of tags, at least the most common ones.
What would you say is the idea way to assign custom attrs?

Reply:
If we just apply them for a small and restricted list of tags, it has no real benefit.

I have a small idea on how to implement this correctly, i'll post it soon in the custom attribute issue.

Reply:
Another way to do this  systematically is to add a wrapper to htmlAttributes to so that it can access more context, then it will know what kind of tag it is, and is there a custom-attr-[tag] for the subject tag, if so, it can attach the attr to it. This will require a bigger refactor. Let me know what you think about this approach.

Reply:
I think that can be done too, but then it still require to add them 1 by 1 let's continue that conversation in #200 👍